acts

```diff
@@ -149,7 +149,7 @@ archives=$($tarsnap --list-archives | sort -n)
 # PART 4: Trim old backups.
 
 if [ "$backuprc" != "0" ] ; then
-    log_message "One of the backups failed -- not deleting old backups"
+    log_verbose "at=error type=$archivetype"
```

Comment:
Actually, I also feel this modification removes too much context -- the user would have to read `acts` source to understand what is happening.